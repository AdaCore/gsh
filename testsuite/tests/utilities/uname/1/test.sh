echo "--help"
uname --help

echo ""
echo "no option" # as if -s was specified
unamenoopt=`uname`
echo $unamenoopt

echo ""
echo "option -a"
unamea=`uname -a`
echo $unamea

echo ""
echo "option -m"
uname -m

echo ""
echo "option -n"
uname -n

echo ""
echo "option -r"
uname -r

echo ""
echo "option -s"
unames=`uname -s`
test $unamenoopt = $unames; echo $unames " is equiv to option uname " $unamenoopt " (true): " $?

echo ""
echo "option -v"
uname -v

echo ""
echo "options -mnrsv" # must be equiv to option -a
unamemnrsv=`uname -mnrsv`
test $unamea = $unamemnrsv; echo $unamemnrsv " is equiv to uname -a " $unamea " (true): " $?

echo ""
echo "option -p"
uname -p

echo ""
echo "option -i"
uname -i

echo ""
echo "option -o"
uname -o

echo ""
echo "option -moi -p"
uname -moi -p
