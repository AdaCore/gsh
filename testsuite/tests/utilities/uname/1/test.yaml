title: Test for uname builtin
