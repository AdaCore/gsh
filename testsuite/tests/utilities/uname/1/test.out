--help
usage: uname [OPTIONS]

no option
CYGWIN_NT-6.0-WOW64

option -a
CYGWIN_NT-6.0-WOW64 machine 1.7.0(0.212/5/3) 2009-09-11 01:25 i686 Cygwin

option -m
i686

option -n
machine

option -r
1.7.0(0.212/5/3)

option -s
CYGWIN_NT-6.0-WOW64  is equiv to option uname  CYGWIN_NT-6.0-WOW64  (true):  0

option -v
2009-09-11 01:25

options -mnrsv
CYGWIN_NT-6.0-WOW64 machine 1.7.0(0.212/5/3) 2009-09-11 01:25 i686  is equiv to uname -a  CYGWIN_NT-6.0-WOW64 machine 1.7.0(0.212/5/3) 2009-09-11 01:25 i686 (true):  0

option -p
unknown

option -i
unknown

option -o
Cygwin

option -moi -p
i686 unknown unknown Cygwin
